packages/main/src/themes/sap_belize_hcb/GroupHeaderListItem.less

```diff
@@ -1,87 +1,11 @@
 /* ============================= */
-/* Base HCB theme                    */
+/* Base HCB theme                */
 /* ============================= */
 @import "../base/GroupHeaderListItem.less";
 @import "../sap_belize/GroupHeaderListItem.less";
 
 /* ============================= */
-/* Global Belize parameters      */
+/* Global Belize HCB parameters  */
 /* ============================= */
 @import "./base.less";
-@import "./global.less";
-
-.sapMGHLI .sapMLIBSelectM,
```

Comment:
Can we remove all these styles? No one uses them?

Reply:
Part of them are already declared in the base and some such as:
sapMLIBSelectS, sapMLIBSelectD, sapMLIBSelectD, sapMListTblSelCol, sapMGHLICounter, sapMListTblUnreadCol are never applied.

I tested in Chrome, Firefox, Safari and IE before and after the removal and I could not find a difference.